### ISSUE TITLE
could you add css3 "filter" support?

### ISSUE THREAD

**Issue by mapleLeat** [danfickle/openhtmltopdf#285]

**Comment by rototor** [comment]
I don't think that this is technically possible. If you only want the filter-style on images you can just alter the images yourself using the existing java libraries for image manipulation. For everything else there is just no support in the PDF standard, as it has no shader system which would allow such filters.

**Comment by rototor** [comment]
I don't think that this is technically possible. If you only want the filter-style on images you can just alter the images yourself using the existing java libraries for image manipulation. For everything else there is just no support in the PDF standard, as it has no shader system which would allow such filters.
